examples/basic-express-typescript/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "basic-express-typescript",
+  "version": "0.0.1",
+  "description": "Example of using TypeScript with fh-sync",
+  "main": "server.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "ts-node server.ts"
+  },
+  "license": "MIT",
+  "dependencies": {
+    "bluebird": "~3.5.1",
+    "body-parser": "~1.18.2",
+    "cors": "~2.8.4",
+    "express": "~4.16.2"
+  },
+  "devDependencies": {
+    "@types/bluebird": "~3.5.18",
+    "@types/body-parser": "~1.16.8",
+    "@types/cors": "~2.8.3",
+    "@types/express": "~4.0.39",
+    "ts-node": "^3.3.0"
```

Comment:
@wtrocki You don't want to make this change to ts-node. Or at least you definitely don't want to remove typescript as a dependency. Right now, ts-node will use whatever version of typescript is installed globally, and there's no guarantee that it's a compatible version. (Also, I don't think it's common to use ts-node on prod apps, so this wouldn't be ideal as a starting point.)

Reply:
Good point. Reason why this worked for me is that I had typescript installed from previous npm install. I forgot that ts-node requires typescript to be available globally. 
I made that change as original change failed to start: `tsc compiler is not available`. 
Before making any change - @evanshortiss What aproach do you prefer?

Reply:
I don't have typescript installed locally so `npm start` failed for me. I installed it locally via package.json (like what Evan had) and `npm start` worked fine, so pretty sure that's all that's necessary. 

What version do you have installed globally? I think the missing types issue that my other comment was about might have to do with that, as I was able to revert your changes and it compiled fine. (And intellisense was correct.)